components/spi/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
+  - https://github.com/redhat-appstudio/service-provider-integration-operator/config/default?ref=d7a6475c6d948452cdc00941616024c9ef6607cc
```

Comment:
We'll need https://github.com/redhat-appstudio/service-provider-integration-operator/commit/4f8d0b4d4e9980518f9a759f7683639e025bc32c ?